script.js

```diff
@@ -1,406 +1,399 @@
-    image: './recipe-images/grilled.jpg'
-  }
+    url: "http://www.seriouseats.com/recipes/2012/05/spice-rubbed-grilled-flap-meat-sirloin-tip-recipe.html",
+    image: "./recipe-images/grilled.jpg",
+  },
+  {
+    name: "Gnocchi Alla Sorrentina",
+    cuisineType: ["italian"],
+    ingredients: [
+      "2 tablespoons extra-virgin olive oil",
+      "1 clove garlic, lightly crushed and peeled",
+      "Generous pinch dried red pepper flakes",
+      "4 cups passata or tomato purée",
+      "Fine salt",
+      "Two sprigs basil",
+      "1 ball mozzarella, diced",
+      "16 ounces potato gnocchi, homemade or store-bought",
+      "Freshly grated Parmigiano-Reggiano",
+    ],
+    source: "Domenica Marchetti",
+    totalTime: 60,
+    url: "https://www.themediterraneandish.com/gnocchi-alla-sorrentina/",
+    image: "./recipe-images/gnochi.jpeg",
+    favorite: true,
+  },
 ]
+
+//Function to show the recipes
+const showRecipes = (recipesToShow) => {
+  container.innerHTML = ""
+
+  recipesToShow.forEach((recipe) => {
+    // Capitalize the first letter of cuisineType
+    const capitalizedCuisineType =
+      recipe.cuisineType[0].charAt(0).toUpperCase() +
+      recipe.cuisineType[0].slice(1)
+
+    container.innerHTML += `
+    <div class="recipe-card">
+      <img src="${recipe.image}"/>
+      <div class="recipe-text">
+        <h3>${recipe.name}</h3>
+        <p><b>Cuisine:</b> ${capitalizedCuisineType}</p>
+        <p><b>Cooking time:</b> ${recipe.totalTime} minutes</p>
+        <p><b>Ingredients:</b></p>
+        <ul>
+          ${recipe.ingredients
+            .map((ingredient) => `<li>${ingredient}</li>`)
+            .join("")}
+        </ul>
+        <p class="recipe-link"><a href="${
+          recipe.url
+        }" target="_blank">Link to cooking method</a>
+        </p>
+      </div>
+    </div>
+    `
+  })
+}
+
+//Initial rendering of all recipes
+showRecipes(recipes)
+
+//Function random
+const getRandomValue = () => {
+  container.innerHTML = ""
+
+  recipes[Math.floor(Math.random() * recipes.length)]
+  const recipe = recipes[Math.floor(Math.random() * recipes.length)]
+  container.innerHTML += `
+  <div class="recipe-card">
+    <img src="${recipe.image}"/>
+    <div class="recipe-text">
+      <h3>${recipe.name}</h3>
+      <p><b>Cuisine:</b> ${recipe.cuisineType}</p>
+      <p><b>Cooking time</b>: ${recipe.totalTime} minutes</p>
+      <p><b>Ingredients:</b></p>
+      <ul>
+        ${recipe.ingredients
+          .map((ingredient) => `<li>${ingredient}</li>`)
+          .join("")}
+      </ul>
+      <p class="recipe-link"><a href="${
+        recipe.url
+      }" target="_blank">Link to cooking method</a>
+      </div>
+  </div>
+  `
+}
+
+//Function to show favorites
+const showFavorites = () => {
+  const favRecipes = recipes.filter((recipe) => recipe.favorite === true)
+  showRecipes(favRecipes)
+}
+
+//Function to sorting descending
```

Comment:
I believe that the Ascending and Descending should be reversed when looking at the result on your site

Reply:
Fixed - thanks for noticing 🤩